src/main/java/com/uid2/shared/cloud/TaggableCloudStorage.java

```diff
@@ -0,0 +1,8 @@
+package com.uid2.shared.cloud;
+
+import java.util.Map;
+
+public interface TaggableCloudStorage extends ICloudStorage {
+
+    void setTags(String cloudPath, Map<String, String> tags) throws CloudStorageException;
```

Comment:
Will we set object tags after they have been uploaded? Could we add a parameter for tags in the upload object method?